spec/lib/middleware/websocket/controller_spec.rb

```diff
@@ -69,4 +69,19 @@
       expect(subject.first).to eq(controller.clients[1])
     end
   end
+
+  describe '#update_all_clients' do
+    subject { controller.update_all_clients }
+    #let(:obj) { double(:write) }
+
+    it 'calls the write method on each client' do
+      controller.on_open(incoming_client_1)
+      controller.on_open(incoming_client_2)
```

Comment:
Would be more "correct" (as far as RSpec is concerned) if we performed this setup in a `before` block.

Reply:
fixed.